README.md

```diff
@@ -10,4 +10,32 @@
 * The game asks to enter players nickname before the start.
 * Card images are generated randomly
 * The image of the covered card is generated randomly (but it should look like the ordinary playing card back).
-* After revealed all the cards congratulations and the time the game took will pop up.
\ No newline at end of file
+* After revealed all the cards congratulations and the time the game took will pop up.
+
+## Development mode
+
+### Install project dependencies:
+
+`yarn` or `npm install`
+
+Installs all necessary dependencies.
```

Comment:
Usually it's 
```
1 Some description
`some command`
2 Some other description
` some other command`
```
No need to write extended description twice like in line 17 and 21 which pretty much mean the same.